tests/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  php-cli:
+    working_dir: /app
+    build:
+      context: ..
+      dockerfile: ./tests/docker/php/cli/Dockerfile
+    restart: on-failure
+    volumes:
+      - ./runtime:/app/tests/runtime
+    depends_on:
+      - redis
+  redis:
+    image: redis:latest
```

Comment:
it would be great to fix the minimal version

Reply:
Write the minimum available version from hub.docker.com: 3.0-alpine